.lighthouserc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "ci": {
+    "collect": {
+      "startServerCommand": "npm run serve",
+      "startServerReadyPattern": "Server running at http://localhost:3000/",
+      "url": [
+        "http://localhost:3000/en",
```

Comment:
I'm not a fan of checking hardcoded URLs.

Let's check all pages besides of blog-posts. Also, do you know how the report gets generated?

We were adding a bot that would comment on the PR?

Reply:
I was also looking for a way to not hardcode the URLs. Check this link to get the full view of how the reports gets generated : https://developer.chrome.com/docs/lighthouse/overview/